helix-core/src/main/java/org/apache/helix/HelixParticipantProperty.java

```diff
@@ -0,0 +1,70 @@
+package org.apache.helix;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.util.Properties;
+import org.apache.helix.model.CloudConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * hold participant properties
+ */
+public class HelixParticipantProperty {
+  private static final Logger logger = LoggerFactory.getLogger(HelixParticipantProperty.class.getName());
+  private String _version;
+  private Integer _healthReportLatency;
```

Comment:
I have no idea what this is for since there are no descriptions, but we usually use `long` for latency values?